graylog2-web-interface/src/views/components/Search.tsx

```diff
@@ -115,7 +115,11 @@ const useAutoRefresh = (refresh: () => Promise<unknown>) => {
   }), [refresh]);
 };
 
-const Search = () => {
+type Props = {
+  InfoBarSlot?: React.ComponentType,
```

Comment:
Usually all props start lowercase.

Reply:
It starts from uppercase because we expect React component.